helm-chart-sources/pulsar/templates/proxy/proxy-ws-configmap.yaml

```diff
@@ -30,11 +30,18 @@ metadata:
     cluster: {{ template "pulsar.fullname" . }}
 data:
   {{- if .Values.proxy.disableZookeeperDiscovery }}
+  {{- if .Values.proxy.useStsBrokersForDiscovery }}
```

Comment:
We should add this value to the default `values.yaml` file since that is where we document features right now.

Also, I wonder if there are any other locations we should add this configuration so that non-statefulset users don't have their deployments break on upgrade.

Reply:
I will add this to values. 

Btw, it doesn't seem to be a new feature as another if stm was using it in the proxy-configmap.yaml: https://github.com/datastax/pulsar-helm-chart/blob/master/helm-chart-sources/pulsar/templates/proxy/proxy-configmap.yaml#L33.

Reply:
I have tested the changes with tlsEnabled set to false and `proxy.useStsBrokersForDiscovery: false`. I couldn't see any issue with that. So, I think upgrading a non-sts deployment to the newer hc version shouldn't be an issue either.

Reply:
@MMirelli - I didn't realize this feature was already present, thank you for clarifying. I see it now.